Gds/bin/osx/terminal-wrapper.bash

```diff
@@ -2,7 +2,7 @@
 ####
 # terminal-wrapper.bash:
 #
-# Terminal wrapper wraps the call to the OSX Terminal.app to run the application in an external
+# Terminal wrapper wraps the call to the macOS Terminal.app to run the application in an external
```

Comment:
Apple has gone through a number of branding iterations... `macOS` is the current brand. `OSX` was never correct.